lib/fortnox/api/models/order.rb

```diff
@@ -19,6 +19,13 @@ class Order < Fortnox::API::Model::Base
 
         # OrderRows Separate object
         attribute :order_rows, Array[OrderRow]
+
+        # Override to_hash to convert nested OrderRows to hash correctly.
+        def to_hash *args
```

Comment:
Did you try without this? In my would to_hash should be recursive by default, but I might well be wrong...